current/en-us/6. routing/1. configuration.md

```diff
@@ -1174,7 +1174,12 @@ export class App {
 
 ## Reusing an Existing View Model
 
-Since the view model's navigation lifecycle is called only once, you may have problems recognizing that the user switched the route from `Product A` to `Product B` (see below). To work around this issue implement the method `determineActivationStrategy` in your view model and return hints for the router about what you'd like to happen. Available return values are `replace` and `invoke-lifecycle`. Remember, "lifecycle" refers to the navigation lifecycle.
+For route transitions across different view-models, aurelia will always create a new instance of the new view model. But when the same view-model is used for two routes (see below), the view model is not replaced by default and only the view model's navigation lifecycle is called. You may have problems recognizing that the user switched the route from Product A to Product B. To work around this issue, you can implement the method `determineActivationStrategy` in your view model. This will return hints for the router about what you'd like to happen. Available return values are:
+
+* replace
+* invoke-lifecycle.
+
+`Replace` does not re-use the existing view-model, i.e. you get a new instance of product. This is less performant but if your view model builds up local state, you will have a clean start.`Invoke-lifecycle` means, invoke the navigation lifecycle. Aurelia then re-uses the View-Model and (only) issues canDeactivate, deactivate, canActivate, activate callbacks on the same instance - e.g. new Product() - when switching from one route to another. This is efficient, but might surprise developers, because the view-model instance is not in a clean state.
```

Comment:
Pls correct the values to be the correct string used in code, fix `Replace` to `replace`, and `Invoke-lifecycle` to `invoke-lifecycle`.

Reply:
There is a missing space before `` `invoke...`` too.

Reply:
Thx for the help. I did so.